blockmanager.go

```diff
@@ -221,6 +248,28 @@ func (b *blockManager) resetHeaderState(newestHash *wire.ShaHash, newestHeight i
 	}
 }
 
+// peerInfo returns info about the peer which is relevant to the blockmanager.
+func (b *blockManager) peerInfo(p *peer.Peer) (*peerInfo, error) {
+	b.peersMtx.RLock()
+	defer b.peersMtx.RUnlock()
+
+	pInfo, ok := b.peers[p]
+	if !ok {
+		return nil, errors.New("missing peer info")
```

Comment:
Please include some more context with this error.  Should it be hit, this error would be pretty useless for debugging.  Something like:

```Go
return nil, fmt.Errorf("missing peer info for %s", p)
```